theme/less/_navigation.less

```diff
@@ -1,25 +1,103 @@
-nav
+// The main navigation menu
+
+#main-navigation 
 {
-  ul
+  background: linear-gradient(#fcfcfc, #ececec);
+}
+
+#main-navigation > ul
+{
+  height:50px;
```

Comment:
Detail: Let's either reuse this height for `top ` in the `#main-navigation > ul ul` selector or remove this here, I would rather reuse it, because it will prevent errors of hardcoding hack we have discussed.

Consdering that we are not planing to apply this property to whole website, like general colors which we keep in `variables.less`. You can try making use of [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/var).

In this case we might have something like:
```
#main-navigation
{
  --menu-height: 50px;
}
```
And use it by `height: var(--menu-height);`

Note: When/if you move the mobile code into current `<nav>` the example might need to be updated.

Reply:
Here I removed `height` option and left menu height to control padding of anchors. Is it Ok?

Reply:
It's okey, we can revisit this issue, when we face a problem and/or during enhancements.